db/migrate/20221020073535_create_motorcycles.rb

```diff
@@ -0,0 +1,13 @@
+class CreateMotorcycles < ActiveRecord::Migration[7.0]
+  def change
+    create_table :motorcycles do |t|
+      t.string :model
+      t.integer :duration
+      t.text :description
+      t.integer :price
+      t.references :user, null: false, foreign_key: { to_table: :users }
+
```

Comment:
Would be Nice to have an image column for the in the Motocycle table to store selected images